app/src/main/java/com/llamalabb/TTT3D/tictactoe/BoardRecyclerAdapter.kt

```diff
@@ -12,18 +12,19 @@ import timber.log.Timber
 /**
  * Created by andy on 11/5/17.
  */
-class BoardRecyclerAdapter(private val presenter: TicTacToeContract.AdapterPresenter)
+class BoardRecyclerAdapter(private val boardListener: TicTacToeContract.BoardListener
+)
     : RecyclerView.Adapter<BoardRecyclerAdapter.CellViewHolder>() {
 
     override fun getItemCount(): Int {
-        return presenter.getCellCount()
+        return boardListener.getCellCount()
     }
 
     override fun onBindViewHolder(holder: CellViewHolder, position: Int) {
-        presenter.onBindCellAtPosition(position, holder)
+        boardListener.cellBinding(position, holder)
```

Comment:
cellBinding here is a noun. When you have a method that is not a constructor, you are doing something and so should use a verb or the `on_____` format.